templates/takeovers/_openstack_security_takeover.html

```diff
@@ -0,0 +1,79 @@
+<section class="p-strip--dark p-takeover js-takeover">
+  <div class="row">
+    <div class="p-takeover__body u-clearfix u-vertically-center">
```

Comment:
Please put these classes on the row above. A row needs to be the direct parent of the cols to remove the first columns right margin.